tests/periph_flashpage_unittest/main.c

```diff
@@ -32,6 +32,10 @@
 
 #include "periph/flashpage.h"
```

Comment:
This literally just defines `tests_flashpage(void)` which you rename to `main(void)` here. So it can be removed (though Hauke might be happy if you move his `@author` tag here).